addons/barcodes/models/barcode_rule.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import re
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class BarcodeRule(models.Model):
+    _name = 'barcode.rule'
+    _order = 'sequence asc'
+
+
+    def _get_type_selection(self):
```

Comment:
yomma miyaaaaaa......
I think my morning is really very cool..... :laughing: 
Hey bro. @rpi-odoo which standers you are use for this coding ???
If i am wrong then correct me but tell me the name of the standards which supports after start class, define global variable then need to take 2 blank line....... ???
![anigif_enhanced-28576-1412437486-1](https://cloud.githubusercontent.com/assets/14940179/12079881/5a48979e-b206-11e5-81c7-2fe9b55832ad.gif)
@morning found your clone :laughing: